spark/deploy/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
@bruwozniak I would add the image name as comment in the first line example:

`# pubnative/spark-base`